### DISCUSSION TITLE
Adding local rule to filter out noise

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#1850] [Unsupported Versions] [answered]
Hello, 

I have a distributed deployment with a manager node, and 2 heavy nodes. I'm looking to add a local rule to suppress broadcast traffic (255.255.255.255) from our IP cameras. I followed documentation from Security Onion site, but still having trouble. I tried putting into: /opt/so/saltstack/local/pillar/global.sls, and /opt/so/saltstack/local/salt/idstools/local.rules using BPF.  I tried to disable by using:

Steno:
 BPF
  - not dst host 255.255.255.255

Can someone help me?

**Comment by weslambert** [comment]
To be clear, are you:

- trying to filter **all** of the traffic at a global level
- trying to filter for PCAP/Zeek/Suricata only

**Reply by kennysden** [reply]
I believe Suricata only. In our Kibana dashboard, we have so many logs with broadcast traffic and I was want to get rid of it.

**Comment by shipler** [comment]
maybe you should disable the noisy suricata rules sids?

**Comment by kennysden** [comment, marked as answer]
I figured it out. I added a BPF rule: !(port 8080 and broadcast)

**Reply by DinhDiesel** [reply]
just curious was the format like this?

nids:
 bpf:
  - not 255.255.255.255 && 8080
